migration/migrator/migrations/system/20240712022346_libreoffice_py

```diff
@@ -0,0 +1,21 @@
+"""Migration for the Submitty system."""
+import os
+
+def up(config):
+    """
+    Run up migration.
+
+    :param config: Object holding configuration details about Submitty
+    :type config: migrator.config.Config
+    """
+    os.system("apt-get install libreoffice")
```

Comment:
Add a `-y`. Also libreoffice is a pretty large package due to all the dependencies. I'm assuming most of the functionality and dependencies we will never use. There looks to be a core or non-gui version which we can install which might be smaller. Do you remember how big the install of libreoffice was for you.

Reply:
I initially tried the nogui and core options, but both failed on the convert command due to some missing package(s) so I switched to the full bundle. I think it was ~600mb for me. I'll test again (=once the rest of the issues on this PR are addressed and see if theres a smaller set of packages we can install.